README.md

```diff
@@ -21,20 +20,16 @@ wget https://github.com/snapp-incubator/thanos-federate-proxy/releases/download/
 tar xvzf thanos-federate-proxy-${VERSION}.linux-amd64.tar.gz thanos-federate-proxy-${VERSION}.linux-amd64/thanos-federate-proxy
 ```
 
-
 ### From source
 
-```
+```sh
 git clone https://github.com/snapp-incubator/thanos-federate-proxy
 cd thanos-federate-proxy && go build
 ./thanos-federate-proxy <optional-extra-flags>
 ```
 
-
-
 ## Configuration
 
-
 Flags:
 
 ```
```

Comment:
```suggestion
```text
```
as markdown code blocks should have language specified

Reply:
Also, please add a todo for adding the markdown lint to CI

Reply:
done.